addons/calendar/data/mail_template_data.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <odoo>
-    <data noupdate="1">
+    <data>
```

Comment:
what? You can't remove a `noupdate` 😆 That'll override user changes on every module update.